app/Services/Html/Concerns/Forms.php

```diff
@@ -56,6 +56,24 @@ public function tags(string $type): Select
             ->value($this->model->tagsWithType($type)->pluck('name', 'name'));
     }
 
+    public function associatedModelsSelect(string $name, string $nameProperty = 'name', ?iterable $options = null)
```

Comment:
It's only about one model right? so `model` is better than `models`. `related` which is used in the code, also sounds better than `associated`.

So let's rename this to `relatedModelSelect`

Reply:
It is actually possible to associate multiple models (`data-select='tags'` is used to display the multiple models). Will rename the methods to `relatedModelsSelect`.